ansible/roles/logging/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,33 @@
+########################### Filebeat Configuration ###########################
```

Comment:
Why do we have this and `ansible/roles/logging/files/filebeat.yml`?

Reply:
This is the one that will be used in the actual Ansible deployment; the static yml file is exactly identical to how the template is rendered in "dev mode" (e.g. deploying to that logging-test.kb.se box that I've been using). It's left here because docker-compose.yml needs to be able to include it and I didn't come up with a better place for hiding it... But we could also pass it in as bart of the "run filebeat" command-line, might be a bit hard to read/maintain though. Do you have any preference?

Reply:
I'd say keep it as is. It's always scary when you don't have a single source of truth, but I hope we'll manage. :)